locales/en.yaml

```diff
@@ -15,90 +15,150 @@ system:
 
               {{ .Commands }}
           cancel:
-            help: Cancel the current activated operation
-            alreadyCancelledAll: There is no activated operation to cancel
+            help: Cancel the current operation
+            alreadyCancelledAll: There is no operation to cancel
 
 modules:
   telegram:
     chatMigration:
-      notification: |
-        {{.Name}} @{{.Username}} 监测到您的群组已从 <b>群组（group）</b> 升级为了 <b>超级群组（supergroup）</b>，届时，群组的 ID 将会发生变更，<b>现已自动将过去的历史记录和数据留存自动迁移到了新的群组 ID 名下</b>，之前的设置将会保留并继续沿用，不过需要注意的是，由于 Telegram 官方的限制，迁移事件前的消息 ID 将无法与今后发送的消息 ID 相兼容，所以当下一次总结消息时将不会包含在迁移事件发生前所发送的消息，由此带来的不便敬请谅解。
+  notification: |
+    {{.Name}} @{{.Username}} has noticed your group's upgrade to a <b>supergroup</b>, which has resulted in a change of the group ID. Rest assured, we have seamlessly migrated all historical data to the new ID, with all your settings preserved intact. Due to Telegram's restrictions, however, message IDs from before the upgrade will not correspond with those sent afterwards, meaning they will be omitted from future summaries. We apologize for any inconvenience this adjustment may cause.
+
+welcome:
+  messageSuperGroup: |
+    🤗 Welcome to @{{.Username}}!
+
+    - Use /smr@{{.Username}} <code>article link</code> for web article summaries.
+
+    - For chat history summaries, assign me as admin (with or without permissions) using a <b>recognizable identity</b> to avoid errors. Then, initiate /configure_recap@{{.Username}} to set up chat recap.
```

Comment:
```suggestion
    - For chat history summaries, assign me as admin (with or without permissions) using a <b>non-anonymous identity</b> to avoid permission detection errors. Then, initiate /configure_recap@{{.Username}} to set up chat recap.
```